config.yaml

```diff
@@ -0,0 +1,25 @@
+cloud:
```

Comment:
Is this an example config? If so, the file name should contain this information and be placed in the appropriate directory.

Reply:
Yes, this is a configuration example. Your suggestion has been received, thank you very much

Reply:
Yes, this is a configuration example. Your suggestion has been received, thank you very much